### ISSUE TITLE
Support LiveReload refreshes so that it doesn't kill the extension

### ISSUE THREAD

**Issue by allejo** [skmail/responsive-viewer#29]
I'm not sure how feasible it would be for this extension to intercept reload requests from LiveReload, but it'd be awesome if a refresh didn't require the need to reenable the extension. For example, I'm building a Jekyll website with the `--livereload` option and it isn't do a "hot reload" but instead a full reload of the page.

**Comment by skmail** [comment]
Hi, @allejo thanks for your message & support.

can you please provide a minimal git repository for your Jekyll build, I want to test it to be able to fix the issue.

I've tested the responsive viewer with a couple of projects that have hot-reloading using NextJS and create-react-app. and it worked fine. 

Thank you.

**Comment by allejo** [comment]
Yea, I've used this extension before on other projects that used hot-reloading and it handled it just fine! However, Jekyll doesn't have hot-reloading and instead triggers a full refresh of the page.

Here's a demo repo for a plain Jekyll website, https://github.com/allejo/responsive-viewer-29

https://user-images.githubusercontent.com/1246453/119463352-8b4d2e80-bcf6-11eb-9785-423846014764.mov

**Comment by dotnetshadow** [comment]
> 
> 
> Hi, @allejo thanks for your message & support.
> 
> can you please provide a minimal git repository for your Jekyll build, I want to test it to be able to fix the issue.
> 
> I've tested the responsive viewer with a couple of projects that have hot-reloading using NextJS and create-react-app. and it worked fine.
> 
> Thank you.

Hi there, firstly a great extension.

I'm trying to get this working with nuxt js, I've noticed that while the pages come up the first time, if I make changes to my nuxt app, no refresh occurs. I have to manually hit the reload button in repsonsive viewer to see the updated content.

I'm using nuxt static option with content. So I think I'm facing the same issue as @allejo 

Just wondering if something has changed recently, or it never supported this scenario?

**Comment by skmail** [comment]
Hi @allejo  We've released a new update to fix the livereload issue, sorry it took too long to get enough free time to find a solution for this issue. we hope it fixes your current problem.

**Comment by skmail** [comment]
@dotnetshadow than for reporting this issue, I Will have a look at this issue and fix it, but my initial investigation shows that opening few tabs for a nuxt app causes the hot reloading to get stuck and not responding to changes, will do more research and if that something can be fixed from the responsive viewer side, then will fix it.

**Comment by allejo** [comment]
Unfortunately, it seems like this issue still persists 😦

**Comment by skmail** [comment]
@allejo  is the extension got updated to v1.0.10 ?

**Comment by dotnetshadow** [comment]
@skmail Yeah I'm facing the same issue even with the updated extension v1.0.10. Basically if I open my localhost site using `npm run dev` and make a chance in my code I can see nuxt trying to update with it's indicator [80%...100%] but the page doesn't refresh in the browser.

**Comment by allejo** [comment]
Oh my gosh, I am so sorry @skmail! I guess my extension hadn't been updated? It's working correctly with Jekyll for me now! :partying_face: :heart:
